SFPortfolioApp/SFPortfolioApp/Modules/CategoryModule/BaseListScene/CategoryTableViewCell/CategoryItemTableViewCell.swift

```diff
@@ -30,6 +31,11 @@ class CategoryItemTableViewCell: UIResizableTableViewCell {
         }
     }
     
+    private func setupAccessibility() {
+        tagsStackView.isAccessibilityElement = true
+        tagsStackView.accessibilityIdentifier = "tagsStackView"
```

Comment:
Extract to the Constants struct or create an enum only for the accessibilityIdentifiers